app/src/main/java/com/pps/globant/fittracker/mvp/presenter/LoginPresenter.java

```diff
@@ -29,19 +29,18 @@
 
 public class LoginPresenter {
 
-    private final LoginModel model;
-    private final LoginView view;
-    private final Activity activity;
-
     //GOOGLE
     /*GOOGLE_SERVICE_CLIENT_ID is a key obtained from https://developers.google.com/identity/sign-in/android/start-integrating
     To get it, it requires an unique SHA1 key, so if you want to recompile the app in another pc, you'll need to create a new key.*/
     public static final String GOOGLE_SERVICE_CLIENT_ID = "602409589834-6b03kf79u58uel71s3k840mjie1mrhm9.apps.googleusercontent.com";
+    public static final String EXTRA_MESSAGE = "com.pps.globant.fittracker.USERID";
     private static final int RC_GET_TOKEN = 9002;
-
+    private final LoginModel model;
+    private final LoginView view;
+    private final Activity activity;
```

Comment:
remove this, and use view.getActivity()

Reply:
into the Francisco's previous PR i think you suggested to do this way, the activity is final and is initialized with view.getActiity(). is just to not call view.getActivity() every time

Reply:
@nacho-globant

Reply:
never mind, i already noticed that it is a strong reference to the activity and can lead to memory leaks